x-pack/plugins/apm/server/routes/apm_routes/get_global_apm_server_route_repository.ts

```diff
@@ -38,7 +38,7 @@ import { eventMetadataRouteRepository } from '../event_metadata/route';
 import { suggestionsRouteRepository } from '../suggestions/route';
 import { agentKeysRouteRepository } from '../agent_keys/route';
 
-const getTypedGlobalApmServerRouteRepository = () => {
+function getTypedGlobalApmServerRouteRepository() {
```

Comment:
I don't know why you changed this but thank you!! :D

Reply:
that makes two of us!